guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -84,4 +75,11 @@ ifeval::["{context}" == "{smart-proxy-context}"]
 ----
 endif::[]
 
-For more information about configuring DHCP, DNS, and TFTP services, see {ProvisioningDocURL}Configuring_Network_Services_provisioning[Configuring Network Services] in _{ProvisioningDocTitle}_.
+.Additional resources
+* For more information about the `{installer-scenario}` command, enter `{installer-scenario} --help`.
+ifeval::["{context}" == "{smart-proxy-context}"]
+ifndef::foreman-deb,orcharhino[]
+* For information about configuring DNS, DHCP, and TFTP externally, see xref:configuring-external-services[].
+endif::[]
+* For more information about configuring DHCP, DNS, and TFTP services, see {ProvisioningDocURL}Configuring_Network_Services_provisioning[Configuring Network Services] in _{ProvisioningDocTitle}_.
+endif::[]
```

Comment:
Lines 80-85 are only included in Installing_Proxy (note the ifeval on line 80). They are not included in Installing_Server and Installing_Server_Disconnected. This is because Installing_Proxy currently includes only proc_configuring-dns-dhcp-and-tftp.adoc, but not proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc, so I didn't create the assembly for Installing_Proxy.

TL;DR: Installing_Server and Installing_Server_Disconnected include these resources only once, on the assembly level. Installing_Proxy includes these resources only once, on the module level.

Reply:
My concern was to not have the "Additional resources" heading twice in a row, but now that I see the diff, I see that there is another module inbetween. So you can ignore this comment.